src/pages/Home/components/ShelterListView/ShelterListView.tsx

```diff
@@ -38,8 +38,7 @@ const ShelterListView = React.forwardRef<HTMLDivElement, IShelterListViewProps>(
         <h1 className="text-[#2f2f2f] font-semibold text-2xl">
           {searchParams.toString()
             ? `Abrigos encontrados (${count})`
-            : `Total de abrigos  (${count})`
-          }
+            : `Total de abrigos  (${count})`}
```

Comment:
Está com um espaço a mais
```suggestion
            : `Total de abrigos (${count})`}
```

Reply:
Perfeitas considerações. Subi no commit 'code review' :)